### ISSUE TITLE
osc-devices vs osc

### ISSUE THREAD

**Issue by atgreen** [zzkt/osc-devices#3]
As I understand it, osc-devices is a superset of osc.  Is that correct?  For the ocicl repo (a quicklisp alternative), I should probably include osc-devices and not osc.  They both provide the osc system.  Do you agree?

**Comment by zzkt** [comment]
yes, osc-devices provides an additional 'devices' interface using clos. There were some regressions introduced when updating the osc protocol from 1.0 to 1.1 hence the fork/split (details can be found at https://github.com/zzkt/osc/issues/18 )

At the moment, I would recommend using zzkt/osc which provides a relatively stable functional style interface. It also provides a simpler base to build other systems (such as osc-devices) so I think it's preferable to use zzkt/osc for any packages installed via quicklisp or ocicl (also, [tests](https://github.com/zzkt/osc/blob/core/osc-tests.lisp))
